Source/Editor/Surface/VisjectSurface.ContextMenu.cs

```diff
@@ -221,7 +221,7 @@ public virtual void ShowSecondaryCM(Vector2 location, SurfaceControl controlUnde
 
             _cmFormatNodesConnectionButton = menu.AddButton("Format node(s)", () =>
             {
-                FormatGraph(SelectedNodes);
+                //FormatGraph(SelectedNodes);
```

Comment:
leftover.

Reply:
Yeah that one for some reason keeps giving an error in VS, got fed up and just commented it out.